src/components/form/DgfipEntrantsTechniques.js

```diff
@@ -144,6 +144,10 @@ const DgfipEntrantsTechniques = ({
           disabled={disabled}
           value={ips_de_production}
         />
+        <p>
```

Comment:
In my opinion, this should look like an helper for the input value with a style similar to this one https://github.com/betagouv/signup-front/blob/882fb29f9b832f40846e47e50be0213904aba29b/src/components/Form.js#L318-L320